src/nxdoc/web-ui/web-ui-tutorials/how-to-create-bulk-edit-form-studio.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'HOWTO: Create a Bulk Edit Form Using Nuxeo Studio'
+review:
+    comment: ''
+    date: '2021-12-21'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create a bulk edit form using Nuxeo Studio Designer
+        level: Intermediate
+        tool: null
+        topics: 'Web UI, Studio Designer, UIElements, Bulk'
+labels:
+    - tutorial
+tree_item_index: 497
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create a form to edit metadata on a large set of documents.
+{{! /excerpt}}
+
+## Requirements
+
+{{#> callout type='info'}}
+This capability is available since LTS 2021 with Web UI 3.0.9.
+{{/callout}}
+
+- A [Contract document type]({{page version='' space='nxdoc' page='web-ui-document-layouts'}}#create-a-contract-document-type) created in Nuxeo Modeler.
+- The Nuxeo [Web UI addon](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-web-ui) installed on your instance.
+- In Studio Modeler > **Settings** > **Application Definition**, make sure that **Nuxeo Web UI** is in the **Packages to Install** list.
+- Activate the `Select All and Bulk Actions` feature by adding the `nuxeo.selection.selectAllEnabled=true` property in your [nuxeo.conf]({{page page="configuration-parameters-index-nuxeoconf"}}) file.
```

Comment:
When Select All becomes widely available and no longer hidden behind a feature flag (LTS 2022?), how can we make sure that this step is no longer presented as a requirement?
And also, is the Select All feature flag strictly necessary here? If not enabled, the feature will still be shown and usable when selecting documents.

Reply:
I agree that you can have bulk edit without select all. Perhaps we should link people to another page (if we have it) where we explain how to enable select all, and let them know that this is an option?

Reply:
I added a mention about that in both pages:

{{#> callout type='tip'}}
Bulk edit can work without enabling the select all option if you intend to select to edit documents manually only.
{{/callout}}

That should be sufficient.